packages/formation-react/src/components/ResponsiveTable/ResponsiveTable.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const borderClasses =
+  'vads-u-border-top--0 vads-u-border-right--0 vads-u-border-left--0 vads-u-font-family--sans vads-u-padding--0 vads-u-padding-y--0p5 medium-screen:vads-u-padding--1';
+const rowPaddingClass = 'vads-u-padding-y--2';
+
+function ResponsiveTable(props) {
+  const { currentSort, fields, data, ariaLabelledBy } = props;
+
+  return (
+    <table aria-labelledby={ariaLabelledBy} className="responsive" role="table">
+      <thead>
+        <tr role="row">
+          {fields.map((field) =>
+            field.nonSortable ? (
+              <th key={field.value}>{field.label}</th>
+            ) : (
+              <th
+                key={field.value}
+                className={borderClasses}
+                role="columnheader"
+                scope="col"
+              >
```

Comment:
I'm not sure why the header for a `nonSortable` field has different classes, role, and scope than a sortable one. It may make more sense to just change whether or not we render the label inside of the button.